synapseclient/client.py

```diff
@@ -1024,6 +1024,7 @@ def store(self, obj, *, createOrUpdate=True, forceVersion=True, versionLabel=Non
         properties, annotations, local_state = split_entity_namespaces(entity)
         bundle = None
         # Anything with a path is treated as a cache-able item
+        properties['versionComment'] = properties['versionComment'] if 'versionComment' in properties else None
```

Comment:
Can you add a comment here that explains why we are doing this, e.g.
```Explicitly set an empty versionComment property if none is supplied, otherwise an existing entity bundle's versionComment will be copied to the update.```

Reply:
Can you also move the above comment ```Anything with a path is treated as a cache-able item``` below this line, it seems like it applies to the next line not this line.